docs/_posts/murat-gunay/2021-12-06-roberta_token_classifier_icelandic_ner_is.md

```diff
@@ -0,0 +1,133 @@
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model, imported from Hugging Face, was fine-tuned on the MIM-GOLD-NER dataset for the Icelandic language, leveraging `Roberta` embeddings and using `RobertaForTokenClassification` for NER purposes.
+
+## Predicted Entities
+
+`Date`, `Location`, `Miscellaneous`, `Money`, `Organization`, `Percent`, `Person`, `Time`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/roberta_token_classifier_icelandic_ner_is_3.3.2_2.4_1638796728651.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")\
+       .setInputCols(["document"])\
+       .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+      .setInputCols(["sentence"])\
+      .setOutputCol("token")
+
+tokenClassifier = RoBertaForTokenClassification.pretrained("roberta_token_classifier_icelandic_ner", "is"))\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+model = nlpPipeline.fit(empty_data)
+text = """Ég heiti Peter FERGUSSON. Ég hef búið í New York síðan í október 2011 og unnið hjá Tesla Motor og þénað 100K $ á ári."""
+result = model.transform(spark.createDataFrame([[text]]).toDF("text"))
+```
+```scala
+val documentAssembler = DocumentAssembler()
+      .setInputCol("text")
+      .setOutputCol("document")
+
+val sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")
+       .setInputCols(Array("document"))
+       .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()
+      .setInputCols(Array("sentence"))
+      .setOutputCol("token")
+
+val tokenClassifier = RoBertaForTokenClassification.pretrained("roberta_token_classifier_icelandic_ner", "is"))\
+  .setInputCols(Array("sentence","token"))\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(Array("sentence", "token", "ner"))\
+      .setOutputCol("ner_chunk")
+      
+val pipeline = new Pipeline().setStages(Array(documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter))
+
+val example = Seq.empty["Ég heiti Peter FERGUSSON. Ég hef búið í New York síðan í október 2011 og unnið hjá Tesla Motor og þénað 100K $ á ári."].toDS.toDF("text")
```

Comment:
Fergusson. And as we discussed last week, it's better to have an example from WIki. I know it takes longer to find a good one, but with these examples we can't make sure it's correct.